stdlib/public/core/Arrays.swift.gyb

```diff
@@ -849,7 +849,7 @@ public struct ${Self}<Element>
 
   @_semantics("array.get_element")
   @inline(__always)
-  public // @testable
```

Comment:
The idea was that we might get such a feature—and there is one, spelled a bit differently, though it only pierces through to internal. But since its implementation is "make this public from this module's perspective" we decided it wasn't a realistic test of the standard library—too many optimization losses by making *everything* internal public. So Dmitri et al went with these notes instead about why things were marked public.